code/src/index.css

```diff
@@ -5,9 +5,85 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  color: white;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.Container{
+  border: solid black;
+  margin: 25px;
+  background-color: black; 
+  display: flex;
+  justify-content: center;
+}
+
+
+.img-container{
+  border: black;
+  position: relative;
+}
+
+
+p{
+   color:gainsboro;
+}
+
+.album-wrapper {
+display: flex;
+flex-wrap: wrap;
+}
+
+h1 {
```

Comment:
Might I suggest a margin to the H1 so the text becomes more centered and close to the design?